src/java/main/org/apache/zookeeper/ZooKeeperMain.java

```diff
@@ -273,7 +275,7 @@ protected void connectToZK(String newHost) throws InterruptedException, IOExcept
         host = newHost;
         boolean readOnly = cl.getOption("readonly") != null;
         if (cl.getOption("secure") != null) {
-            System.setProperty(ZooKeeper.SECURE_CLIENT, "true");
+            System.setProperty(ZKClientConfig.SECURE_CLIENT, "true");
```

Comment:
I don't think using it is much of a problem because once you remove it, the compiler will complain about the dangling references. But to be honest, I'm not stuck on the idea of doing such assignments, I'm just trying to be safe regarding future changes before we remove the deprecated references.